translations/de/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,169 @@
+# German translations for PROJECT.
```

Comment:
minor: not sure where this came from but looks a bit weird/generic.

Reply:
👀 no idea how this got here, should not indeed